README.md

```diff
@@ -1,24 +1,235 @@
-# README
+<div align="center">
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <img src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.crunchbase.com%2Forganization%2Fmicroverse&psig=AOvVaw0WLXyWBbF0vEkXZK7Osr4E&ust=1670482418477000&source=images&cd=vfe&ved=0CA8QjRxqFwoTCND2_ov25vsCFQAAAAAdAAAAABAE" alt="logo" width="140"  height="auto" />
+  <br/>
 
-Things you may want to cover:
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
```

Comment:
- [ ] Kindly ensure [your frontend repo](https://github.com/aneallaryea100/hello-react-front-end/pull/1) has proper documentation using [the new README format](https://github.com/microverseinc/readme-template)

- [ ] Kindly ensure the [frontend repo README](https://github.com/aneallaryea100/hello-react-front-end/pull/1) has link to this backend project

Reply:
- Front end repo has proper documentation using new README format
- Front end repo has link to backend project